packages/examples/react-demo/src/SignHelpers.ts

```diff
@@ -91,13 +91,13 @@ export const send_personal_sign = async (provider: SDKProvider) => {
     const message = 'Hello World from the Create React dapp!';
     const hexMessage = '0x' + Buffer.from(message, 'utf8').toString('hex');
 
-    const sign = await window.ethereum.request({
+    const sign = await window.ethereum!.request({
```

Comment:
in this case also might be better to check before and print error if window.ethereum not present but overall might be better to have example using the provider

Reply:
seems like it was using the old `window.ethereum` approach. Switched to the same as sign_v4